prod-nais.yaml

```diff
@@ -24,7 +24,7 @@ spec:
     - name: GSAK_URL
       value: https://oppgave.nais.adeo.no/api/v1/oppgaver
     - name: ENABLE_GSAK
-      value: "false"
-prometheus:
-  enabled: true
-  path: /kontakt-oss-api/internal/actuator/prometheus
+      value: "true"
```

Comment:
Skal denne stå på som default?

Reply:
Nå har vi jo egentlig slått dette på for oss selv, og jeg synes det burde reflekteres i master. Det er kjipt hvis vi ved et uhell skrur av gsak fordi vi har glemt å sette denne manuelt til "true" før deploy